src/features/user/Account/EditModal.tsx

```diff
@@ -225,21 +223,23 @@ export const EditModal = ({
                     >
                       <Controller
                         render={(properties) => (
-                          <DatePicker
+                          <MuiDatePicker
```

Comment:
Calendar styles are different, which is OK. Only selected date colors may need some adjustment (esp. the font color)
![Screenshot 2022-05-05 at 5 27 51 PM](https://user-images.githubusercontent.com/44917347/166918478-d4ff5c06-fb3e-4d32-94a7-f1f691db7f58.png)

Reply:
@mohitb35 MUI-date-pickers comes from [@mui/lab](https://www.npmjs.com/package/@mui/lab).
Components in Mui lab are not completely stable. So for now Mui Core doesn't provide a standard way to style these components.

Reply:
https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1466#discussion_r867484958

Reply:
Issue created #1486